src/xarray_regrid/methods/conservative.py

```diff
@@ -44,16 +50,27 @@ def conservative_regrid(
     Args:
         data: Input dataset.
         target_ds: Dataset which coordinates the input dataset should be regrid to.
+        latitude_coord: Name of the latitude coordinate. If not provided, attempt to
+            infer it from the first coordinate containing the string 'lat'.
+        skipna: If True, enable handling for NaN values. This adds some overhead,
+            so should be disabled for optimal performance on data without NaNs.
+        nan_threshold: Threshold value that will retain any output points containing
+            at least this many non-null input points. The default value is 1.0,
+            which will keep output points containing any non-null inputs. The threshold
+            is applied sequentially to each dimension, and may produce different results
+            than a threshold applied concurrently to all regridding dimensions.
 
     Returns:
         Regridded input dataset
     """
-    if latitude_coord is not None:
-        if latitude_coord not in data.coords:
-            msg = "Latitude coord not in input data!"
-            raise ValueError(msg)
-    else:
-        latitude_coord = ""
+    if latitude_coord is None:
+        for coord in data.coords:
+            if "lat" in coord.lower():
+                latitude_coord = coord
+                break
+    if latitude_coord not in data.coords:
+        msg = "Latitude coord not in input data!"
+        raise ValueError(msg)
```

Comment:
I would not want to raise an error if there is no latitude coordinate: currently the regridding package is still quite generic and can be used to regrid any data with euclidean coordinates.